### DISCUSSION TITLE
Possible to replace images?

### DISCUSSION THREAD

**Discussion by visualsitemaps** [GoogleChromeLabs/ProjectVisBug#600] [Q&A] [answered]
i looked everywhere.. expecting this basic feature. but sadly i was unable to find it.
does it exist?

p.s. amazing work! many tough UX issues for non designer newbies to onboard. 
Consider making a simpler version for marketers.

**Comment by visualsitemaps** [comment, marked as answer]
ok i see that if you finally select the <img>object ( hard to do when it is wrapped around other classes btw ) then you can drag a new image into it.

**Reply by argyleink** [reply]
yep, the feature exists and can even handle uploading multiple images! thanks for asking and starting the discussion here 🙂

selection, especially of nested elements, is a tricky aspect of working with web sites in general. to aid with this, the `enter` key dives deeper into a selection (if it can), while also `shift + enter` will go up the layers/elements tree. this is crucial for selecting nested elements that a mouse can't reach. small additional tidbit, you can use `tab` and `shift+tab` to move laterally. you can navigate any and every element with the keyboard, where the mouse cant!

so a typical image replacement workflow may go:
1. point and click the image
2. hit enter a few times (to ensure you're selecting the deepest element which will be an img) until you notice VisBug isnt selecting new elements
3. drag in your new image and watch it swap

you can also use the search feature to find <img> tags: `.post img` for example will find all img tag elements inside of a .post
